// uuid/dist/esm-browser/index.js
C:\Windows\System32\novaintroducao-nodejs\node_modules\uuid\dist\esm-browser\index.js
export { default as v1 } from './v1.js';
^^^^^^

SyntaxError: Unexpected token 'export'
←[90m    at wrapSafe (internal/modules/cjs/loader.js:1060:16)←[39m
←[90m    at Module._compile (internal/modules/cjs/loader.js:1108:27)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1164:10)←[39m
←[90m    at Module.load (internal/modules/cjs/loader.js:993:32)←[39m
←[90m    at Function.Module._load (internal/modules/cjs/loader.js:892:14)←[39m
←[90m    at Module.require (internal/modules/cjs/loader.js:1033:19)←[39m
←[90m    at require (internal/modules/cjs/helpers.js:72:18)←[39m
    at Object.<anonymous> (C:\Windows\System32\novaintroducao-nodejs\node_modules\←[4msequelize←[24m\lib\utils.js:7:16)
←[90m    at Module._compile (internal/modules/cjs/loader.js:1144:30)←[39m
←[90m    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1164:10)←[39m
PS C:\Windows\System32\novaintroducao-nodejs>export { default as v1 } from './v1.js';
export { default as v3 } from './v3.js';
export { default as v4 } from './v4.js';
export { default as v5 } from './v5.js';
export { default as NIL } from './nil.js';
export { default as version } from './version.js';
export { default as validate } from './validate.js';
export { default as stringify } from './stringify.js';
export { default as parse } from './parse.js';
